src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -136,6 +139,9 @@ export class QuestionSubmissionFormComponent implements OnInit {
    * Triggers the change of the recipient submission form.
    */
   triggerRecipientSubmissionFormChange(index: number, field: string, data: any): void {
+    this.isQuestionValid.subscribe((isValid: boolean) => {
```

Comment:
To answer your question:
- We don't need to do listen to validation event here. There are better places.
- Plus: This is wrong. You are subscribing the observable multiple times.